test/Microsoft.ML.Functional.Tests/ModelLoading.cs

```diff
@@ -220,25 +248,33 @@ public void SaveCompositeLoaderAndLoad()
             var model = composite.Fit(file);
 
             string modelPath = GetOutputPath(FullTestName + "-model.zip");
-            _ml.Model.Save(model, modelPath);
+            _ml.Model.Save(null, model, modelPath);
```

Comment:
should this be `_ml.Model.Save(model, null, modelPath);` ?

Reply:
No in this case the test is calling the loader model. I agree it's confusing but fixing up naming in a test was not my priority.

Reply:
OK, got some time, changed the name here... now it's `loaderWithEmbeddedModel`.